Documentation/metrics.md

```diff
@@ -1,46 +1,18 @@
 # Metrics
 
-**NOTE: The metrics feature is considered experimental. We may add/change/remove metrics without warning in future releases.**
+etcd uses [Prometheus][prometheus] for metrics reporting. The metrics can be used for real-time monitoring and debugging. etcd only keeps these data in memory. If a member restarts, metrics will be reset.
```

Comment:
s/etcd only keeps these data in memory. If a member restarts, metrics will be reset./etcd does not persist its metrics; if a member restarts, the metrics will be reset./